tests/unit_tests/test_deplete_activation.py

```diff
@@ -47,7 +47,7 @@ def model():
     ("direct", {}, 1e-5),
     ("flux", {'energies': ENERGIES}, 0.01),
     ("flux", {'energies': ENERGIES, 'reactions': ['(n,gamma)']}, 1e-5),
-    ("flux", {'energies': ENERGIES, 'reactions': ['(n,gamma)'], 'nuclides': ['W186']}, 1e-5),
+    ("flux", {'energies': ENERGIES, 'reactions': ['(n,gamma)'], 'nuclides': ['W186', 'H3']}, 1e-5),
```

Comment:
Really subtle point here as to why this was failing with a 1e-5 tolerance. In the test below, we do an initial run to determine the capture rate and use that to predict what source rate would reduce the material by half. However, when we add additional nuclides from the flux helper for the actual depletion simulation, we end up getting a different answer for the capture rate (within statistics) because loading data for another nuclide _even when it doesn't show up in a material_ will change the random number ordering due to this line:
https://github.com/openmc-dev/openmc/blob/7200d426440c4fa14dc71610ede5eac96ddf785a/src/physics.cpp#L154

Bumping up the tolerance here should get the test to pass:
```suggestion
    ("flux", {'energies': ENERGIES, 'reactions': ['(n,gamma)'], 'nuclides': ['W186', 'H3']}, 1e-2),
```